.github/ISSUE_TEMPLATE/UX Test

```diff
@@ -0,0 +1,15 @@
+## Driving Questions ##
+*What are we hoping to validate?*
+
+
+
+## Testing Plan ##
+
+- [ ] Use Checklist for Tasks
+
+## Additional context ##
+*Screenshot or link to what needs to be tested*
+
+Related to issue #1310 
```

Comment:
Are all of these going to be related to issue 1310?

Reply:
Oops no - just need that to say related to - I will remove